README.md

```diff
@@ -39,7 +39,83 @@ make test
 ./scripts/tests --help
 ```
 
+## Formal Verification
+
+We use the [C Bounded Model Checker (CBMC)](https://github.com/diffblue/cbmc) to prove absence of various classes of undefined behaviour in C, including out of bounds memory accesses and integer overflows. The proofs cover all C code in [mldsa/*](mldsa) and [mldsa/fips202/*](mldsa/fips202) involved in running mldsa-native with its C backend. See [proofs/cbmc](proofs/cbmc) for details.
+
+## Security
+
+All assembly in mldsa-native is constant-time in the sense that it is free of secret-dependent control flow, memory access,
+and instructions that are commonly variable-time, thwarting most timing side channels. C code is hardened against compiler-introduced
+timing side channels through suitable barriers and constant-time patterns.
+
+Absence of secret-dependent branches, memory-access patterns and variable-latency instructions is also tested using `valgrind`
+with various combinations of compilers and compilation options.
```

Comment:
The tests using valgrind are not yet implemented. And also we still have some issues open re constant-time propertier. We cannot put this paragraph in as of now.
Please remove.